kserve/base/inferenceservice-config-patch.yaml

```diff
@@ -4,17 +4,7 @@ metadata:
   name: inferenceservice-config
   namespace: kserve
 data:
-  explainers: |-
-    {
-        "alibi": {
-            "image" : "$(kserve-alibi-explainer)",
-            "defaultImageVersion": "$(kserve-alibi-explainer-version)"
-        },
-        "art": {
-            "image" : "$(kserve-art-explainer)",
-            "defaultImageVersion": "$(kserve-art-explainer-version)"
-        }
-    }
+  explainers: "{}"
```

Comment:
`kustomize build` shows 
~~~
 deploy: |-
    {       
      "defaultDeploymentMode": "Serverless"
    }       
  explainers: '{}'
  ingress: |-   
    {             
        "ingress.....
~~~
I didn't test it but I am not sure if `{}` is ok to deploy kserve and a model? If it works, I am ok with it for now before we integrate with TrustyAI.

Reply:
We should be safe if we always omit using explainers.

As soon as we try to use some explainer, we may have issues. But we are not supporting them at the moment.